src/main/java/com/pinterest/secor/io/KeyValue.java

```diff
@@ -25,16 +25,29 @@
  */
 public class KeyValue {
 	
-	private final long mKey;
+	private final long mOffset;
+	private final byte[] mKey;
```

Comment:
I think we should renamed the old 'mKey' to 'mKafkaKey' (so getKey() would become getKafkaKey()) this way we wouldn't run into the danger of confusing the usage of the 'getKey()' in one of the caller site.